spec/models/lettings_log_derived_fields_spec.rb

```diff
@@ -0,0 +1,1178 @@
+require "rails_helper"
+require "shared/shared_examples_for_derived_fields"
+
+# rubocop:disable RSpec/BeforeAfterAll
+# rubocop:disable RSpec/InstanceVariable
+RSpec.describe LettingsLog, type: :model do
+  before(:context) do
+    owning_organisation = build(:organisation)
+    @log = build(:lettings_log, :startdate_today, owning_organisation:, managing_organisation: owning_organisation)
```

Comment:
Any reason we're not using let to define log? Looks like defining it this way in specs is an anti-pattern (?)

Reply:
this was required by doing before(:context) but I got rid of that now